witwidget/notebook/colab/wit.py

```diff
@@ -70,7 +70,7 @@ def compute_custom_distance(wit_id, index, callback_name, params):
     (function() {{
       const id = {id};
       const wit = document.querySelector("#wit");
-      wit.parentElement.style.height = '{height}px';
+      wit.style.height = '{height}px';
```

Comment:
why was this change necessary?

Reply:
discussed offline, due to how wit.html contains WIT